substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/reports/AnalysisReportsOptions.java

```diff
@@ -52,6 +53,15 @@ protected void onValueUpdate(EconomicMap<OptionKey<?>, Object> values, Boolean o
         }
     };
 
+    @Option(help = "Change the output format of the analysis call tree. See: Reports.md.")//
```

Comment:
Since you are referring to `Reports.md` it would be nice to mention this option in the `#### Call tree` section there.

Reply:
From that help message the user would not know that the two possible option values are `TXT` and `CSV`.